<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2UM13OP" failures="1" tests="1" name="Test" time="13.297" errors="0" timestamp="2025-04-22T23:41:15 IST">
  <testcase classname="com.coverFoxTest.CoverFoxTestUsingTestNGFail" name="validateCoverFoxPlansFail" time="0.034">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.coverFoxTest.CoverFoxTestUsingTestNGFail.validateCoverFoxPlansFail(CoverFoxTestUsingTestNGFail.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateCoverFoxPlansFail -->
</testsuite> <!-- Test -->
